src/index.html

```diff
@@ -2,16 +2,211 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link 
+      rel="stylesheet" 
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action = "https://mate-academy-form-lesson.herokuapp.com/create-application" method = "post">
+
+      <fieldset class = "container personal-information">
```

Comment:
```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">

      <fieldset class="container personal-information">
```

Reply:
You didn't check and fix this issue in all places. Please, make sure to check all your code and fix that in all places

Reply:
Here we go again, you didn't fix the comment